src/libYARP_dev/src/devices/ServerInertial/ServerInertial.cpp

```diff
@@ -33,7 +33,7 @@ yarp::dev::DriverCreator *createServerInertial()
 /**
  * Constructor.
  */
-yarp::dev::ServerInertial::ServerInertial() : ownDevices(false), subDeviceOwned(NULL)
+yarp::dev::ServerInertial::ServerInertial() : IMU_polydriver(YARP_NULLPTR), ownDevices(false), subDeviceOwned(NULL)
```

Comment:
not very important, but there is a typo in the commit log, poiter -> pointer 😆

Reply:
I've modified it... :sweat_smile: